vscode-trace-extension/src/utils/backend-tsp-client-provider.ts

```diff
@@ -67,9 +69,16 @@ export async function updateNoExperimentsContext(): Promise<void> {
     return;
 }
 
-async function getExternalUriFromUserSettings(): Promise<vscode.Uri> {
+async function getExternalUriFromUserSettings(isFrontEnd: boolean): Promise<vscode.Uri> {
     const tsConfig = vscode.workspace.getConfiguration('trace-compass.traceserver');
-    const traceServerUrl: string = tsConfig.get<string>('url') || 'http://localhost:8080';
+    let traceServerUrl: string;
+    if (isFrontEnd) {
+        traceServerUrl = tsConfig.get<string>('url') || 'http://localhost:8080';
+    } else {
+        traceServerUrl = tsConfig.get<string>('enableSeparateBackendUrl')
```

Comment:
should this be "<boolean>" instead of "<string>"?

Reply:
Fixed